spec/better_errors/error_page_spec.rb

```diff
@@ -88,5 +88,35 @@ def initialize(message = nil)
         error_page.exception_message.should_not =~ /\A\n\n/
       end
     end
+
+    context 'with an inspect size limit set' do
+      before { BetterErrors.maximum_variable_inspect_size = 50_000 }
```

Comment:
Seeing that you override this limit in the last example, the `context` description is a bit misleading. Would be too  verbose if we set the limit inside each example?